app/assets/stylesheets/responsive/_new_request_style.scss

```diff
@@ -121,6 +121,10 @@ input#reedit_button {
   border: none;
 }
 
+.request_deny_response {
+  background-color: #FFF;
+}
+
 
```

Comment:
Headache-based nitpick!

I found the results - particularly with the longer question in place - a bit hard on the eyes when it came to seeing where the question ends and the form begins so could we add something like this...

```
.js-loaded {
  div.request_form_response--allow {
    div#request_subject {
      margin-top: 1em;
      padding-top: 1em;
      border-top: 1px solid #ccc;
      border-color: rgba(0, 0, 0, 0.1);
    }
  }
}
```

... to add a divider between the sections (assumes that subject is the first input in the form, only applies when JS is loaded and the form is nested inside a `request_form_response--allow` block)

Before:

![screen shot 2019-01-23 at 17 13 25](https://user-images.githubusercontent.com/27760/51624395-a5ae4c00-1f32-11e9-92c5-8ec72151149e.png)

After:

![screen shot 2019-01-23 at 17 12 39](https://user-images.githubusercontent.com/27760/51624408-acd55a00-1f32-11e9-8fda-8f3447333a47.png)

Reply:
Added in 7ffb5b1